wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/AccountViews/TeamAccountView.swift

```diff
@@ -65,6 +55,11 @@ final class TeamAccountView: BaseAccountView {
         }
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        selectionView.layer.cornerRadius = 6
```

Comment:
Isn't it sufficient to set this value once?

Reply:
I would say so. Didn't find any issues during testing.